src/main/scala/com/wavesplatform/matcher/model/OrderHistoryStorage.scala

```diff
@@ -9,7 +9,7 @@ import scorex.transaction.assets.exchange.AssetPair
 class OrderHistoryStorage(db: MVStore) {
   val orders: util.Map[String, String] = db.openMap("orders")
   val ordersInfo: util.Map[String, String] = db.openMap("ordersInfo")
-  val pairAddressToOrderIds: util.Map[AssetPairAddressKey, Set[String]] = db.openMap("pairAddressToOrderIds")
+  val pairAddressToOrderIds: util.Map[AssetPairAddressKey, Array[String]] = db.openMap("pairAddressToOrderIds")
```

Comment:
consider using `compositeKey` if you need to access to "last 50 orders" or similar